.github/workflows/builds.yml

```diff
@@ -421,7 +421,7 @@ jobs:
       - name: ccache stats
         run: ccache -s
       - name: Unit tests
-        run: ${SETUP} && ${PRELOAD} && cmake --build build --target test
```

Comment:
@gagnonlg do you remember why this was necessary? it segfaulted because the `std::filesystem` because of standard lib mismatch and removing solved the problem

Reply:
thanks! looks like it was added here https://github.com/acts-project/acts/pull/871

Reply:
I see! I don't quite remember, but I see no problem with the proposed chabge